Bucks beta ps

GraemeMcCubbin opened the pull request:
- Add the Bucks CC Beta project story to production
- Replace the Bucks CC Discovery and Alpha project story tile with the Bucks CC Beta project story tile on the homepage
- Add the Bucks CC Beta project story tile to the Project Stories index page

Commit:
Add Bucks CC Beta project story

Commit:
Copy and image fix

GraemeMcCubbin committed:
Replace Bucks CC first project story with Bucks CC Beta project story

Replace the Discovery & Alpha Bucks CC project story tile with the new
Bucks CC Beta project story tile on the home page.

mybits commented on source/project-stories/bucks-cc-beta.erb:
```diff
@@ -0,0 +1,210 @@
+  client: "Buckinghamshire County Council"
+  introduction: "Digitalising key local government services with Buckinghamshire County Council: The Beta phase"
+  image: "/project-stories/bucks-cc-beta/header.jpg"
+  image_description: ""
```
Let's put some content in here, otherwise will be missing img `alt`

GraemeMcCubbin replied:
Hey @mybits - thanks for pointing this out, have added in an image description for the header.

mybits commented:
I left one comment re img `alt` attribute missing.
There would also a content cutting here I'm afraid :) :
<img width="659" alt="screen shot 2016-09-26 at 11 45 00" src="https://cloud.githubusercontent.com/assets/5237879/18831727/4677086c-83df-11e6-8903-226f971251cf.png">

And also header image looks a bit blurry on retina. It would be good to change it to higher res if possible.

Otherwise looks good 🌴

GraemeMcCubbin committed:
Add in the Bucks CC Beta project story header description

Add in the image description for the Bucks CC Beta project story header
image.

GraemeMcCubbin committed:
Trim content from Bucks CC Beta project story

Trim content from the Bucks CC Beta project story to prevent the
paragraph from pulling the image up.

GramMcCram commented:
Thanks @mybits - have trimmed the text under that section. Will update the image header at a later time.
